plugin/trino-redshift/pom.xml

```diff
@@ -17,12 +17,28 @@
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
     </properties>
 
+    <repositories>
+        <repository>
+            <id>redshift</id>
+            <url>http://redshift-maven-repository.s3-website-us-east-1.amazonaws.com/release</url>
```

Comment:
Is this artifact not in maven central? We shouldn’t be referencing third-party repositories in our Pom files. It makes it much harder for organizations that want or need to proxy maven through an internal maven proxy and it introduces potential instability in the builds when those third-party repos are unavailable.

Reply:
This is the official repository to obtain the artifact and i didn't find AWS publishing it to Maven Central too.
I hear your concern though. Would it help if we repackage the driver under io.trino (with no shading, no relocation applied)?

Reply:
> Is this artifact not in maven central?

No, unfortunately not. Here's a link to the instructions from AWS: https://docs.aws.amazon.com/redshift/latest/mgmt/jdbc20-install.html#configure-jdbc20-connection-with-maven